mitosheet/mitosheet/tests/step_performers/test_excel_range_import.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+"""
+Contains tests for Excel Range Import
+"""
+
+import os
+import pandas as pd
+import pytest
+from mitosheet.excel_utils import get_row_and_col_index_from_cell_address
+from mitosheet.tests.test_utils import create_mito_wrapper_dfs
+
+TEST_FILE = "test_file.xlsx"
+TEST_SHEET = 'sheet1'
+TEST_DF_1 = pd.DataFrame({'header 1': [1], 'header 2': [2]})
+TEST_DF_2 = pd.DataFrame({'header 100': [1], 'header 200': [2]})
+
+EXCEL_RANGE_IMPORT_TESTS = [
+    (
+        {'dataframe_1': {'type': 'range', 'range': 'A1:B2'}},
+        [TEST_DF_1]
+    ),
+    (
+        {'dataframe_1': {'type': 'range', 'range': 'A1:B2'}, 'dataframe_2': {'type': 'range', 'range': 'A4:B5'}},
+        [TEST_DF_1, TEST_DF_2]
+    ),
+    (
+        {'a bad dataframe name': {'type': 'range', 'range': 'A1:B2'}, '97 also bad': {'type': 'range', 'range': 'A4:B5'}},
+        [TEST_DF_1, TEST_DF_2]
+    ),
```

Comment:
Some other test ideas 
- [ ] Pass the same dataframe to two range imports 
- [ ] Other range formats, like: A5:A1 and B5:A1
- [ ] Dataframes that have more than just 1 row in them 
- [ ] Check that we handle invalid ranges correctly. For example I don't think we support A:C 
- [ ] On Test_DF_1, what happens if we use the range, A1:G5

Reply:
It literally won't let you write the same DF, unfortunately -- but all others are tests. Good tests - these caught some bugs!